google-cloud-bigtable/src/test/java/com/google/cloud/bigtable/admin/v2/it/BigtableInstanceAdminClientIT.java

```diff
@@ -143,29 +146,25 @@ public void instanceCreationDeletionTest() {
     }
   }
 
-  @Test
-  public void clusterCreationDeletionTest() {
-    Instance currentInstance = client.getInstance(instanceId);
-    assume()
```

Comment:
is this no longer true?

Reply:
it uses a brand new instance from instanceAndClusterCreationDeletionTest, which is always prod